### ISSUE TITLE
Contrast bro

### ISSUE THREAD

**Issue by cinpear** [hackclub/flavortown#1617]
<img width="1096" height="156" alt="Image" src="https://github.com/user-attachments/assets/ee734d40-0d89-4ccf-bd85-9160f38cb9e6" />

<img width="1500" height="663" alt="Image" src="https://github.com/user-attachments/assets/a26f6d03-e60b-4515-af48-08741d245cbf" />

the contrast on the devlog post times is really bad like you literally fail accessibility checks lol
